code/datums/mind.dm

```diff
@@ -1215,6 +1226,42 @@
 				log_admin("[key_name(usr)] has automatically forged objectives for [key_name(current)]")
 				message_admins("[key_name_admin(usr)] has automatically forged objectives for [key_name_admin(current)]")
 
+	else if(href_list["brother"])
+		switch(href_list["brother"])
```

Comment:
You probably want to include a section which can auto generate objectives for the brother team. See the `if("autoobjectives")` stuff above for traitor, or any antag which has that.

Reply:
if anything it should be done in teams not in minds.